### ISSUE TITLE
Connect from localhost

### ISSUE THREAD

**Issue by dduenker** [resin-io/resin-wifi-connect#241]
It appears to me, that the ui server is only bound to the wireless network interface.
It would be nice to have it available locally, when a screen is connected.

**Comment by majorz** [comment]
Right, we have plans for making this available on other interfaces as well. Another related issue is #213.

**Comment by majorz** [comment]
@dduenker actually I misunderstood your question -- you can still access the web server locally - it is just not bound on localhost, but on the gateway address. If you type `http://192.168.42.1/` in your browser you should be able to access the captive portal page.

**Comment by dduenker** [comment]
@majorz Sorry if i did not articulate myself correct: i know, that it would be accessible via the ip `192.168.42.1`, but i would need it to be accessible via `http://localhost/` as well. But i guess it really is related to #213 then. :)
Thanks for your time to answer my question. :)
